tox.ini

```diff
@@ -8,11 +8,13 @@ setenv =
     PYTHONPATH = {toxinidir}/matrix_client
 deps =
     .[test]
+    .[e2e]
     coveralls
 commands=
     coverage run --source=matrix_client setup.py test
     coverage report
     - coveralls
+install_command = pip install --process-dependency-links {opts} {packages}
```

Comment:
I was hoping this would stay a Travis hack, as we will have a proper PyPi package by the time E2E is fully merged and usable. Also, `--process-dependency-links` is deprecated and kinda unsecure (replacing it is still an active pip issue according to what I've found).

Reply:
Changed my mind, I'll add a note :)